index.js

```diff
@@ -19,6 +19,7 @@ function Multer (options) {
 
   this.limits = options.limits
   this.preservePath = options.preservePath
+  this.defParamCharset = options.defParamCharset || 'latin1'
```

Comment:
This default shouldn't need to be passed explicitly, busboy uses latin1 anyway when the defParamCharset passed is undefined.

Reply:
Yes, I know - I added it in case something will change in `busboy` default settings, `multer` will still behave consistently 
@LinusU @bjohansebas @UlisesGascon what do you think? I can change it if it's unnecessary precaution